app-drawer/app-drawer.html

```diff
@@ -535,6 +538,39 @@
         this.fire('app-drawer-overlay-changed', { overlay: this.opened && !this.persistent });
       },
 
+      _openedChanged: function(opened) {
+        var evtName = this.opened ? 'opened' : 'closed';
+
+        this.fire('app-drawer-' + evtName);
+      },
+
+      _onTransitionEnd: function(evt) {
+        var opened = this.opened;
+        var evtName = this.opened ? 'opened' : 'closed';
+
+        this.fire('app-drawer-transitioned-' + evtName);
```

Comment:
The `app-drawer-overlay-changed` event only fires when the scrim finishes transitioning, and the scrim only transitions when the drawer is an overlay (i.e. it is above and partially obscuring the main content). It notably does not fire if the drawer is `persistent`.

If you are not using `persistent`, then this event will probably work for you. If you are, then we might need something else, but I don't want to cross that bridge unless we need to.